packages/popup/src/pages/CreateSeed/NewSeed.js

```diff
@@ -20,7 +20,8 @@ function NewSeed() {
   const [groupNamePlaceholder, setGroupNamePlaceholder] = useState('')
   const hdGroup = useHdAccountGroup()
   useEffect(() => {
-    setGroupNamePlaceholder(`Seed-${hdGroup.length + 1}`)
+    setGroupNamePlaceholder(`${t('seedPrefix')}${hdGroup.length + 1}`)
```

Comment:
create account name with hardcode not i18n

Reply:
done